src/components/Form/Readme.mdx

```diff
@@ -33,33 +37,92 @@ as well as FieldsWrapper which takes a JSON config object and renders the whole
 <Props of={Input} />
 
 <Playground>
-  <Input
-    name="first_name"
-    label="Name"
-    placeholder="First Name"
-    onChange={e => console.log(e.target.name, e.target.value)}
-  />
-  <Input
-    name="last_name"
-    placeholder="Last Name"
-    onChange={e => console.log(e.target.name, e.target.value)}
-  />
-  <Input
-    name="linkedin"
-    placeholder="LinkedIn Profile URL"
-    onChange={e => console.log(e.target.name, e.target.value)}
-    icon={<FiLinkedin />}
-    label="LinkedIn Profile URL"
-  />
-  <Input
-    name="phone"
-    type="tel"
-    placeholder="Phone number"
-    onChange={e => console.log(e.target.name, e.target.value)}
-    icon={<FiPhone />}
-    label="Phone Number"
-    context="danger"
-  />
+  {() => {
```

Comment:
Great docs example!

I saw you left the default margin? Happy to remove it in `Input`, but keep it in `Fields`?

You were right about allowing the parent to do the layout here...

Reply:
Yes I think its fine to leave the margin, knowing we can override it. I liked your argument that if we add a few `Input` elements it should still look good.